main.tf

```diff
@@ -136,7 +136,14 @@ resource "aws_iam_policy" "default_cache_bucket" {
   policy = join("", data.aws_iam_policy_document.permissions_cache_bucket.*.json)
 }
 
+data "aws_s3_bucket" "secondary_artifact" {
+  count  = module.this.enabled ? 1 : 0
+  bucket = var.secondary_artifact_location
```

Comment:
If `var.secondary_artifact_location` is the default then this would fail. Can you update the conditional to check against the default value i.e. `null`?

Reply:
Yikes, good catch.